pep-0517.txt

```diff
@@ -218,6 +241,14 @@ metadata. The directory passed in by the build frontend MUST be
 identical to the directory created by ``get_wheel_metadata``,
 including any unrecognized files it created.
 
+This method only places the *contents* of the wheel file, without actually
+creating the final artifact. This is to allow flexibility in how the result is
```

Comment:
This change is annoying for flit, because it builds the `.whl` file directly, without an intermediate directory containing files in the same layout. Not a showstopper if other people want it this way, but I'm -0.5 on it.